spyder/api/plugin_registration/_confpage.py

```diff
@@ -7,33 +7,33 @@
 """Plugin registry configuration page."""
 
 # Third party imports
-from qtpy.QtWidgets import (QGroupBox, QVBoxLayout, QCheckBox,
-                            QGridLayout, QLabel)
+from pyuca import Collator
+from qtpy.QtWidgets import QVBoxLayout, QLabel
 
 # Local imports
-from spyder.api.plugins import SpyderPlugin
 from spyder.api.preferences import PluginConfigPage
 from spyder.config.base import _
-from spyder.config.manager import CONF
+from spyder.widgets.elementstable import ElementsTable
 
 
 class PluginsConfigPage(PluginConfigPage):
+
     def setup_page(self):
         newcb = self.create_checkbox
         self.plugins_checkboxes = {}
 
         header_label = QLabel(
-            _("Here you can turn on/off any internal or external Spyder plugin "
-              "to disable functionality that is not desired or to have a lighter "
-              "experience. Unchecked plugins in this page will be unloaded "
-              "immediately and will not be loaded the next time Spyder starts."))
+            _("Here you can turn on/off any internal or external Spyder "
+              "plugin to disable functionality that is not desired or to have "
+              "a lighter experience.")
```

Comment:
> Mention first that this can help people to simplify the interface, which actually is a better usage of this pane.

:+1: Makes sense from a flow perspective (though it seems most likely that people will use this to disable troublesome external plugins, as people can just close panes they don't want).

> I think `any` instead of `a` sounds better here (`a` makes this sentence odd to me as a non-native speaker).

Hmm..."any" sounds rather odd to me here as a native speaker, whereas I'm sure sure what else to suggest other than "a" here (without a significant rewrite). Also, "any" isn't really correct if some plugins cannot be disabled, as you can't actually disable disable "any" plugin, only certain ones.

Here's my revised suggestion with your other change and a bit of further cleanup in the phrasing enabled by your re-ordering:

```suggestion
            _("Disable a Spyder plugin (external or built-in) to prevent it "
              "from loading until re-enabled here, to simplify the interface "
              "or in case it causes problems.")
```